src/std/text.ab

```diff
@@ -53,29 +53,29 @@ fun sed_version(): Num {
 /// Replaces all occurences of a regex pattern in the content with the provided replace text.
 ///
 /// Function uses `sed`
-pub fun replace_regex(source: Text, search: Text, replace: Text, extended: Bool = false): Text {
+pub fun replace_regex(source: Text, search: Text, replacement: Text, extended: Bool = false): Text {
     trust {
-        const sed_version = sed_version()
+        const sed_ver = sed_version()
         search = replace(search, "/", "\/")
-        replace = replace(replace, "/", "\/")
-        if sed_version == SED_VERSION_GNU or sed_version == SED_VERSION_BUSYBOX {
+        replacement = replace(replacement, "/", "\/")
+        if sed_ver == SED_VERSION_GNU or sed_ver == SED_VERSION_BUSYBOX {
```

Comment:
I prefer `sed_version`

Reply:
it should `sed_version` instead of `sed_type` like it was before your changes in that file

Reply:
there is still in that file `sed_ver` you have to revert it to `sed_version` in all the file